tests/pass/libc.rs

```diff
@@ -290,10 +290,34 @@ fn test_clocks() {
     assert_eq!(is_error, 0);
 }
 
+fn test_posix_gettimeofday() {
+    let mut tp = std::mem::MaybeUninit::<libc::timeval>::uninit();
+    let tz = std::ptr::null::<libc::timezone>();
+    #[cfg(target_os = "macos")] // `tz` has a different type on macOS
+    let tz = tz as *mut libc::c_void;
+    let is_error = unsafe { libc::gettimeofday(tp.as_mut_ptr(), tz) };
+    assert_eq!(is_error, 0);
+    let tv = unsafe { tp.assume_init() };
+    let secs = tv.tv_sec;
+    assert!(secs > 0);
+    let usecs = tv.tv_usec;
+    assert!(usecs >= 0); // Adjust check as theoretically this could be 0.
```

Comment:
```suggestion
    assert!(tv.tv_sec > 0);
    assert!(tv.tv_usec >= 0); // Theoretically this could be 0.
```
No need to be quite so verbose.